cache.py

```diff
@@ -61,8 +58,10 @@ def _get_cache(self):
     def _update_cache(self, cache):
         if not os.path.exists(self._dir):
             os.makedirs(self._dir)
-        with open(os.path.join(self._dir, self._file), 'w') as outfile:
+        with tempfile.NamedTemporaryFile(dir=self._dir, mode='w') as outfile:
             json.dump(cache, outfile)
+            os.remove(self._cache_file()) #minor race case
```

Comment:
This may raise a `FileNotFoundError` when cache file doesn't exist (i.e. first time the tool is run)

Reply:
You're right, thanks!  I pushed a commit that fixes this.